app/css/MainPage.css

```diff
@@ -83,11 +82,9 @@ main {
 .btn-upload {
   margin-top: 10px;
   padding: 10px 20px;
-  font-size: 16px;

```

Comment:
same problem, I would change the 16px with 1rem like so:
font-size: 1rem;
for better scalability and understandability as it is the same by default